includes/class-wc-gateway-stripe.php

```diff
@@ -361,54 +361,12 @@ public function admin_scripts() {
 	}
 
 	/**
-	 * Payment_scripts function.
-	 *
-	 * Outputs scripts used for stripe payment
+	 * Returns the JavaScript configuration object used on the product, cart, and checkout pages.
 	 *
-	 * @since 3.1.0
-	 * @version 4.0.0
+	 * @return array  The configuration object to be loaded to JS.
 	 */
-	public function payment_scripts() {
+	public function javascript_configuration() {
```

Comment:
I do prefer to be explicit, which is why I went with the longer name. It's not really that cumbersome to write when you have autocomplete anyway. I'd like to at least keep the `javascript_` part, because `js` is relatively ambiguous in this context IMO.

But on that note, which do you think is the best descriptor for this; `javascript_parameters`/`javascript_params` or `javascript_configuration`/`javascript_config`?

Reply:
Hm, not a huge deal but I'd vote for `javascript_params` since "configuration" sort of implies to me that it's just user settings when in fact there are also return URLs, nonces, etc., just everything needed for parameterizing the client – what do you think?

Reply:
Your reasoning makes sense, and `javascript_params` sounds good to me 🙂 

Changed the names in c36fa93b5ca73332d2b2400cb2ee4f8abe78db9b 🙌